src/fsharp/FSharp.Core/seq.fsi

```diff
@@ -179,6 +251,19 @@ namespace Microsoft.FSharp.Collections
         /// is reached it returns a -1 if the first sequence is shorter and a 1 if the second sequence
         /// is shorter.</returns>
         ///
+        /// <example>
+        /// <code lang="fsharp">
+        ///     ([1; 2], [1; 2]) ||> Seq.compareWith (fun a b -> a.CompareTo(b)) // evaluates to 0
+        ///     ([1; 2], [1; 3]) ||> Seq.compareWith (fun a b -> a.CompareTo(b)) // evaluates to -1
+        ///     ([1; 3], [1; 2]) ||> Seq.compareWith (fun a b -> a.CompareTo(b)) // evaluates to 1
+        ///     ([1; 2], [1]) ||> Seq.compareWith (fun a b -> a.CompareTo(b)) // evaluates to -1
```

Comment:
This one evaluates to 1 - check them all :)

Reply:
You know? I copied and pasted the output, so I thought I would not make a mistake. I thought I would not make a mistake copying and pasting. That happened.